src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -117,6 +119,13 @@
      */
     @Override
     public boolean compile() throws IOException, InterruptedException {
+        String subscp = "";
```

Comment:
This could probably be moved to `processClassPath()`?

Reply:
Not sure, as `processClassPath()` is really about the application itself and what it uses on the classpath.
This part is about what native-image needs to be able to do its work.